jbmc/src/jbmc/jbmc_parse_options.h

```diff
@@ -58,7 +58,6 @@ class optionst;
   "(refine-strings)" /* will go away */ \
   "(no-refine-strings)" \
   "(string-printable)" \
-  "(string-max-length):" \
   "(string-max-input-length):" /* will go away */ \
```

Comment:
Why is that option not yet removed?

Reply:
I'll remove that one along with the other deprecated options.